.github/workflows/build.yml

```diff
@@ -82,3 +82,25 @@ jobs:
         resource-dirs: '[]'
         clojars-username: ${{ secrets.CLOJARS_USERNAME }}
         clojars-deploy-token: ${{ secrets.CLOJARS_PASSWORD }}
+  
+  release:
+    runs-on: ubuntu-latest
+    needs: build
+    if: ${{ github.ref_type == 'tag' }}
+    steps:
+    - name: Checkout repository
+      uses: actions/checkout@v3
+
+    - name: Download Bundle Artifact
+      uses: actions/download-artifact@v3
+      with:
+        name: datasim-bundle-artifact-${{ github.sha }}
+
+    - name: Craft Draft Release
+      uses: softprops/action-gh-release@v1
```

Comment:
I just found out there's a v2 of the action-gh-release action ([see changelog](https://github.com/softprops/action-gh-release/blob/master/CHANGELOG.md))

Reply:
On those same lines there's v4 of actions/checkout and actions/download-artifact.

Reply:
Once we update those versions we should cut a v.x.x.x-pre version to ensure the artifact release works.